src/styles/block/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  height: $header-height;
+
+  display: flex;
+  justify-content: space-between;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px #0000000d;
```

Comment:
The color value for the `box-shadow` property should be written in the form of an RGBA color to maintain consistency with the transparency value (0d in this case). Consider using `rgba(0, 0, 0, 0.05)` instead of `#0000000d`.